devicetypes/smartthings/zigbee-motion-detector.src/zigbee-motion-detector.groovy

```diff
@@ -67,11 +68,17 @@ def installed(){
 def parse(String description) {
 	log.debug "description(): $description"
 	def map = zigbee.getEvent(description)
-	if(!map){
+	if (!map) {
 		if (description?.startsWith('zone status')) {
 			map = parseIasMessage(description)
 		} else {
-			map = batteyHandler(description)
+			def descMap = zigbee.parseDescriptionAsMap(description)
+			if (descMap?.clusterInt == zigbee.POWER_CONFIGURATION_CLUSTER) {
+				map = batteyHandler(description)
+			} else if (descMap?.clusterInt == zigbee.IAS_ZONE_CLUSTER && descMap.commandInt != 0x07 && descMap.value) {
+				log.debug "parseDescriptionAsMap: $descMap.value"
```

Comment:
It's better to refactor `parseIasMessage ` and this a bit so you parse the value into a `ZoneStatus`: `def zs = new ZoneStatus(zigbee.convertToInt(descMap.value, 16))` and then pass it into a common function for interpreting the ZoneStatus. That way, whether it receives a ZoneStatusChangeNotification or a ZoneStatus attribute report or read attribute response it will handle the message in the same way (including scheduling the motion stopped event for devices that need it).